test/e2e/pages/moves-dashboard.js

```diff
@@ -13,9 +13,10 @@ class MovesDashboardPage extends Page {
       createMoveButton: Selector('.govuk-button').withExactText(
         'Create a move'
       ),
+      filterContainer: Selector('.app-filter'),
       pagination: {
         previousLink: Selector('.app-pagination__list-item--prev a'),
-        todayLink: Selector('.app-pagination__list-item a').withText('Today'),
+        todayLink: Selector('.app-pagination__list-item:nth-of-type(2) a'),
```

Comment:
I think it might be safer to use a different selector for "This week", then we will ensure that the relevant users are seeing the correct link, ie in outgoing moves it's done by day therefore they will see "Today" and for your new journeys we are expecting them to see "This week".

So I'd say create a new item and test for that in the smoke test:

```javascript
thisWeekLink: Selector('.app-pagination__list-item a').withText('This week'),
```